### ISSUE TITLE
NTSCQT For Android

### ISSUE THREAD

**Issue by LarthLover** [JargeZ/vhs#121]
JargeZ Needs To Make A Android Version Of NTSCQT

**Comment by RyanMagyar777** [comment]
Yes

**Comment by RyanMagyar777** [comment]
I see nothing Mario

**Comment by RayyanMakesVHS** [comment]
No, ntscQT does not directly support Android, as it is a Python application with a GUI for Windows, Linux, and macOS that requires a terminal to run. However, because nstscQT uses a cross-platform UI framework, it may be possible to run it on Android with additional development efforts.

**Comment by georgeirinel1986-hue** [comment]
Jiju

**Comment by vtimessquare6-sudo** [comment]
No it suports on windows instead

**Comment by egordobrij3-jpg** [comment]
.

**Comment by egordobrij3-jpg** [comment]
https://github.com/user-attachments/assets/bb7840d6-910b-4788-ad91-1b51e1a6852e

**Comment by human23** [comment]
> https://github.com/user-attachments/assets/bb7840d6-910b-4788-ad91-1b51e1a6852e

What?...
